src/app/globals.css

```diff
@@ -27,9 +27,13 @@
     @apply text-lg;
     box-sizing: border-box;
   }
+  html {
+    font-size: 100%;
+  }
   body {
     @apply text-black relative;
     text-size-adjust: 100%;
+    font-size: 1.8rem;
```

Comment:
What is the advantage of setting everything except `body` element to 18px and the inside to 16px?